src/main/java/me/lambdaurora/keystrokes/gui/KeystrokesColorConfigScreen.java

```diff
@@ -119,9 +119,9 @@ public void apply_rainbow(boolean newValue)
     {
         this.mod.config.setRainbowText(newValue);
 
-        this.foreground_buttons.forEach(button -> button.active = !newValue);
+        this.foregroundButtons.forEach(button -> button.setActive(true));
         if (this.rainbowSaturationButton != null)
-            this.rainbowSaturationButton.active = newValue;
+            this.rainbowSaturationButton.setActive(true);
```

Comment:
I forgot to mention that his is a bit concern, idk if its right

Reply:
should be setActive(newValue)

Reply:
Pushed a new commit fixing it